### ISSUE TITLE
fgets() must greater than 0 (error from $this->redis->get() )

### ISSUE THREAD

**Issue by dominixz** [joelcox/codeigniter-redis#2]
private function _bulk_reply()
    {

```
    // Get the amount of bits to be read
    $value_length = (int) fgets($this->_connection);

    return fgets($this->_connection, $value_length + 1);

}
```

this function fixed problem =
private function _bulk_reply()
    {

```
    // Get the amount of bits to be read
    $value_length = (int) fgets($this->_connection);

    return @fgets($this->_connection, $value_length + 1);

}
```

**Comment by joelcox** [comment]
Thanks for reporting. Will be fixed in a few mins.

**Comment by joelcox** [comment]
Suppressed an error notice whencalling _bulk_reply() with length of reply equal to 0. Closed by add580bd9e1c30eee21379bf34ac8d2083359600. Thanks for reporting dominixz
